src/app/components/pages/admin-cell/admin-cell.component.html

```diff
@@ -1,5 +1,96 @@
+
+      <div class="form-group">
+        <label for="nb_volunteers_needed" class="form-label">Nombre de partcipants requis:</label>
+        <input class="input" formControlName="nb_volunteers_needed" id="nb_volunteers_needed" type="number" autofocus>
+        <div *ngIf="eventForm.controls['nb_volunteers_needed'].hasError('required')"
+             class="form-text form-text--right form-text--danger">Le nombre de volontaires est obligatoire.
+        </div>
+        <div *ngIf="eventForm.controls['nb_volunteers_needed'].hasError('min')"
+             class="form-text form-text--right form-text--danger">Un nombre negatif de volontaires n'est point possible.
+        </div>
+      </div>
+
+      <div class="form-group">
+        <label for="nb_volunteers_standby_needed" class="form-label">Nombre de remplacants requis:</label>
+        <input class="input" formControlName="nb_volunteers_standby_needed" id="nb_volunteers_standby_needed" type="number" autofocus>
+        <div *ngIf="eventForm.controls['nb_volunteers_standby_needed'].hasError('min')"
+             class="form-text form-text--right form-text--danger">Un nombre negatif de remplacants n'est point possible.
+        </div>
+      </div>
+
+      <div class="form-group" *ngIf="cycles">
+        <label for="cycle_id" class="form-label">Cycle:</label>
+        <select formControlName="cycle_id" id="cycle_id" class="input">
+          <option value="" disabled>Selectionner un cycle</option>
+          <option *ngFor="let cycle of cycles" [ngValue]="cycle.id">{{ cycle.name }}</option>
+        </select>
+      </div>
+
+      <div class="form-group" *ngIf="tasktypes">
+        <label for="task_type_id" class="form-label">Type d'activité:</label>
+        <select formControlName="task_type_id" id="task_type_id" class="input">
+          <option value="" disabled>Choose a state</option>
+          <option *ngFor="let taskType of tasktypes" [value]="taskType.id">{{ taskType.name }}</option>
+        </select>
+      </div>
+
+      <div class="form-group">
+
+        <label for="start_date" class="form-label">Date de début de l'event</label>
+          <input [owlDateTimeTrigger]="dt12" [owlDateTime]="dt12"
+                 formControlName="start_date"  class="input"
+                 id="start_date">
+          <owl-date-time #dt12></owl-date-time>
+
+        <!--<input type="datetime-local" formControlName="start_date" id="start_date" class="input">-->
+        <div class="form-text form-text--right form-text--danger"
+             *ngIf="eventForm.controls['start_date'].hasError('dateStartMissing')">Date de début nécessaire.
+        </div>
+      </div>
+
+      <div class="form-group">
+
+
+
+        <label for="end_date" class="form-label">Date de fin de l'event</label>
+          <input [owlDateTimeTrigger]="dt13" [owlDateTime]="dt13"
+                 formControlName="end_date" class="input"
+                 id="end_date">
+          <owl-date-time #dt13></owl-date-time>
```

Comment:
It seems we can use the `firstDayOfWeek` property of this component to use Monday as the first day (in french format)

https://github.com/DanielYKPan/date-time-picker#properties-for-owl-date-time